### ISSUE TITLE
QA_EXPT-5 & EXPT-6_Procedure_content-is-same

### ISSUE THREAD

**Issue by BSravanthi** [Virtual-Labs/analog-cmos-vlsi-circuit-design-iiith#17]
Defect Description :
In the Procedure page of 5th and 6th experiments of this lab, found that the content is same.

Actual Result :
In the Procedure page of 5th and 6th experiments of this lab, found that the content is same.

Environment :
OS: Windows 7, Ubuntu-16.04,Centos-6
Browsers:Firefox-42.0,Chrome-47.0,chromium-45.0
Bandwidth : 100Mbps
Hardware Configuration:8GBRAM ,
Processor:i5

Attachments:
![qa_oe_avlsi_i15](https://cloud.githubusercontent.com/assets/13479177/26631353/401755b0-4628-11e7-81cf-209846eca567.png)

**Comment by sravanthimodepu** [comment]
Yes.. The 5th & 6th experiments belongs to the same experiment. So the content is same, but the simulations are different. Found the same from the vlab.co.in page.

**Comment by BSravanthi** [comment]
Need to contact lab developer. The issue will be opened till we get the response.
